src/pages/LandingPage/MapComponent.js

```diff
@@ -1,77 +1,71 @@
-import React from 'react';
-import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
-import L from 'leaflet';
-import PropTypes from 'prop-types';
-import originIcon from 'App/assets/images/origin.svg';
-import destinationIcon from 'App/assets/images/destination.svg';
+import React, { useEffect, useState } from "react";
+import { MapContainer, TileLayer } from "react-leaflet";
+import L, { Icon } from "leaflet";
+import PropTypes from "prop-types";
+// import originIcon from "App/assets/images/origin.svg";
+// import destinationIcon from "App/assets/images/destination.svg";
```

Comment:
Good work @Rwandarushya, the bus stops are being correctly displayed on the map with the new react-leaflet version. Small fix to make would be to remove the commented out code if it is not needed

Reply:
Thanks for the feedback, i resolved the above issues